### DISCUSSION TITLE
Can I post special deals on Discussions for products I sell directly related to some Github projects?

### DISCUSSION THREAD

**Discussion by Rembacher** [community/community#128188] [Other Feature Feedback, Questions, & Ideas]
As title states I'm a small business in Australia and sell Solar Inverters and would like to post a special discount for Github members directly related to my Solar Inverter Brand. Is this allowed in Discussions? Thanks

**Comment by ebndev** [comment]
Hi @Rembacher, thanks for being a part of the GitHub Community!

We currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thanks for understanding!

**Reply by Rembacher** [reply]
Hi @ebndev , thank you for your info...and fully understand
